modules/secured-cluster-services-config.adoc

```diff
@@ -69,7 +69,7 @@
 | Tag of `collector` image to use.
 
 | `collector.collectionMethod`
-| Either `CORE_BPF`, `EBPF` (deprecated), or `NO_COLLECTION`.
+| Either `CORE_BPF`, or `NO_COLLECTION`.
```

Comment:
```suggestion
| Either `CORE_BPF` or `NO_COLLECTION`. The `EBPF` option is available only for version 4.4 and earlier.
```

Reply:
If we decide not to change the text, should at least remove the extra comma there.